components/developer/jenkins/Makefile

```diff
@@ -49,4 +49,4 @@ test: $(NO_TESTS)
 
 # Auto-generated dependencies
 REQUIRED_PACKAGES += SUNWcs
-REQUIRED_PACKAGES += runtime/java/openjdk8
+REQUIRED_PACKAGES += runtime/java/openjdk17
```

Comment:
Question: Why doesn't get openjdk17 installed with this?

It's not installed on our build server, even after running gmake publish here:
runtime/java/openjdk11                            11.0.11-2022.0.0.1         i--
runtime/java/openjdk17                            17-2022.0.0.0              ---

Reply:
It probably requires manual installation in that case since there is also a mediator that this change changes.

Reply:
Openjdk should get installed when the updates are installed. REQUIRED_PACKAGES might be overriden for this package